internal/versions/common.go

```diff
@@ -128,19 +129,18 @@ func validateReleaseImageForRHCOS(
 ) error {
 	// Multi is not a valid RHCOS CPU architecture, its sub-architectures are
 	if cpuArchitecture == common.MultiCPUArchitecture {
-		return errors.Errorf("The requested RHCOS version (%s, arch: %s) does not have a matching OpenShift release image", rhcosVersion, cpuArchitecture)
+		return errors.Errorf("The requested RHCOS version (%s, arch: %s) does not have a matching OpenShift release image, there are no valid Openshift versions for architecture %s", rhcosVersion, cpuArchitecture, common.MultiCPUArchitecture)
```

Comment:
I would change this message to something like - `RHCOS CPU architecture should be single arch`